system/liquid-ceph/values.yaml

```
@@ -3,13 +3,10 @@ image:
   tag: latest
   pullPolicy: IfNotPresent
 ceph:
-  endpoint: "" # "http://ceph-rgw.ceph.svc.cluster.local:80"
-  targetZone: ""
-  validPools: "" # regex
-  resourceName: "objectstore-ceph-region-3-size"
   secretName: "" # ceph.conf and keyring file
-  rgwSecretName: "" # AccessKey and SecretKey fields
-  allowedDomains: [] 
+  allowedDomains: []
+  zoneMapping: "" # zone mapping file
```

file is misleading there needs to be yaml there
```suggestion
  zoneMapping: {} # zone mapping yaml
```